__tests__/setup/web/index.html

```diff
@@ -92,27 +93,34 @@
       const PASSTHRU_MIDDLEWARE = store => next => action => next(action);
 
       async function main(options) {
-        let { createDirectLine, createStyleSet, props, setup, storeInitialState = {}, storeMiddleware = PASSTHRU_MIDDLEWARE } = unmarshal(options);
+        let { createDirectLine, createStyleSet, offline, props, setup, storeInitialState = {}, storeMiddleware = PASSTHRU_MIDDLEWARE } = unmarshal(options);
 
         props = unmarshal(props);
 
         if (setup) { await setup(); }
 
         await loadScript('/webchat-instrumented.js');
 
-        const { token } = await retry(async () => {
-          try {
-            const res = await fetch('https://webchat-mockbot.azurewebsites.net/directline/token', { method: 'POST', timeout: 2000 });
-
-            return await res.json();
-          } catch (err) {
-            console.error('Failed to fetch Direct Line token from Mockbot.');
-            console.error(err);
-
-            throw err;
-          }
-        }, 3);
-
+        let directLine;
+        if (offline !== false && !createDirectLine) {
```

Comment:
I don't love this line... If `offline` is `undefined` or `false`, I want it to use Offline Mockbot, but `!undefined` is `true` and `!false` is `true so I cant use `if (!offline) {`... Thoughts?

Reply:
@corinagum @compulim

Reply:
I would add a comment to the html outlining the same thing... to me it's going to confuse someone (me) no matter what, lol

Reply:
William and I decided to change `offline` to `useProductionBot`. If `useProductionBot` is `false` or `undefined`, the tests will use Offline Mock Bot; otherwise, they will run with the Production version of Mock Bot.